webapp/apps/dynamic/views.py

```diff
@@ -786,7 +786,9 @@ def behavior_results(request, pk):
     model = url.unique_inputs
 
     first_year = model.first_year
-    micro = model.micro_sim.unique_inputs
+    micro = (model.micro_sim.unique_inputs if model.micro_sim
+             else TaxSaveInputs())
+
     if (micro.json_text is not None and (micro.json_text.raw_reform_text or
```

Comment:
@lucassz  The bug is in how the test was written. While this works, I think it makes more sense to fix the bug in the test. What you could to fix this is create a `OutputUrl` model instance for the microsimulation run in the test and add it as the `micro_sim` attribute on the `DynamicOutputUrl` instance. 

You can use the `get_taxbrain_model` helper function to create the url instance. One example for how to use it for the micro-sim model is here: https://github.com/OpenSourcePolicyCenter/PolicyBrain/blob/master/webapp/apps/taxbrain/tests/test_views.py#L756-L758.

Does that make sense?

Reply:
Yes, thank you, looking into this now.